version/version.go

```diff
@@ -20,6 +20,8 @@ package version
 // name matches github.com/elastic/beats/v7/dev-tools/mage/settings.go parseBeatVersion
 const defaultBeatVersion = "9.0.0"
 
+var qualifier = ""
```

Comment:
Will we manually update this upon new qualifiers? Why not a const?

Reply:
Its a variable that is set at build time, it can't be const.